src/Plugin.php

```diff
@@ -30,32 +33,27 @@ public function compose() {
 		 */
 		do_action( 'satispress_compose' );
 
-		$htaccess_handler = new Htaccess( $this->cache_path() );
-
 		$package_manager = new PackageManager( $this->cache_path() );
 
-		$basic_auth_request = new Authentication\Basic\Request();
-		$basic_auth_request->load();
-
-		$limit_login_attempts = new Integration\LimitLoginAttempts();
-		$limit_login_attempts->load();
+		// Register hook providers.
+		$this
+			->register_hooks( new I18n() )
```

Comment:
We don't have an `I18n` class (yet).

Reply:
That class comes from [cedaro/wp-plugin](https://github.com/cedaro/wp-plugin/blob/master/src/Provider/I18n.php).